demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -89,25 +89,49 @@ def _get_hooks(pre_commit_config: dict) -> dict:
         hooks = {}
         for repo in pre_commit_config["repos"]:
             for hook in repo["hooks"]:
-                hooks[hook["id"]] = hook
+                hooks[hook["id"]] = {"repo": repo, "hook": hook}
                 # if the hook has a skip key, we add it to the SKIPPED_HOOKS set
                 if hook.pop("skip", None):
                     SKIPPED_HOOKS.add(hook["id"])
         return hooks
 
+    def handle_python2_files(self, unit_test: bool) -> List[str]:
```

Comment:
`handle` doesn't clearly explain what this method does. Why does it return a list of strings?
Either rename or explain why it returns a list of strings

Reply:
I don't see a point to return the python2 files